application/app/Http/Requests/API/PrivilegeListRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Requests\API;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Auth;
+
+class PrivilegeListRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return Auth::hasPrivilege('VIEW_PRIVILEGE');
```

Comment:
Yea, that was an oversight. The Privilege key should actually be in some library, so it could be shared between services.
I didn't want to leave some endpoint without privilege, which is why I probably added this authorization check there